src/community/mapml/pom.xml

```diff
@@ -36,6 +41,11 @@
       <groupId>org.geoserver</groupId>
       <artifactId>gs-wms</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.locationtech.proj4j</groupId>
+      <artifactId>proj4j</artifactId>
+      <version>1.0.0</version>
```

Comment:
Two things here:

- we keep version numbers in the root pom.xml in a dependency management section (if not we get madness)
- geotools includes its own referencing implementation so proj4j should not be necessary (why is it being used?)

Reply:
proj4 is used because I had existing code that used it.  I will have a look to see about refactoring the dependency to be geotools instead of proj4j.  Also I'll remove the ${project.version} reference.

Reply:
Okay, most anything you need to do with projections can be accomplished using the [CRS utility class](http://docs.geotools.org/latest/userguide/library/referencing/crs.html) and [JTS](https://docs.geotools.org/latest/userguide/library/main/jts.html) utility class.

Reply:
> we keep version numbers in the root pom.xml in a dependency management section (if not we get madness)

I found that gs-web-demo is not in the dependency management section, but there is a <gs.version> in the <properties> section, so is it ok if I just use ${gs.version} instead of ${project.version} ? I didn't want to mess with the GeoServer pom.xml :-).

Reply:
because gs-web-demo is being built as part of this project it does not need to be defined as part of the dependency management section.

All the other sections use  ${project.version}:

```xml
   <dependency>
      <groupId>org.geoserver</groupId>
      <artifactId>gs-main</artifactId>
      <version>${project.version}</version>
    </dependency>
```

No need to define a custom property since maven already provides one for the current build.